import_3dm/converters/__init__.py

```diff
@@ -60,15 +60,25 @@ def convert_object(context, ob, name, layer, rhinomat, view_color, scale, option
         data = RHINO_TYPE_TO_IMPORT[ob.Geometry.ObjectType](context, ob, name, scale, options)
 
     if data:
-        if ob.Geometry.ObjectType == r3d.ObjectType.InstanceReference:
-            blender_object=data
-        else:
-            data.materials.append(rhinomat)
-            blender_object = utils.get_iddata(context.blend_data.objects, ob.Attributes.Id, ob.Attributes.Name, data)
-            blender_object.color = [x/255. for x in view_color]
-
-    if blender_object:
-        try:
-            layer.objects.link(blender_object)
-        except Exception:
-            pass
\ No newline at end of file
+        data.materials.append(rhinomat)
+        blender_object = utils.get_iddata(context.blend_data.objects, ob.Attributes.Id, ob.Attributes.Name, data)
+    else:
+        blender_object = context.blend_data.objects.new(ob.Attributes.Name+"_Instance", None)
```

Comment:
Instead of `ob.Attributes.Name` i'd suggest using `name` here. The latter holds the instance definitions name (see line 232 in read3dm.py), so instances are called by their block names.